docs/reference/transform/apis/get-transform-stats.asciidoc

```diff
@@ -28,10 +28,14 @@ Retrieves usage information for {transforms}.
 [[get-transform-stats-prereqs]]
 ==== {api-prereq-title}
 
-* If the {es} {security-features} are enabled, you must have `monitor_transform` 
-cluster privileges to use this API. The built-in `transform_user` role has these 
-privileges. For more information, see <<security-privileges>> and 
-<<built-in-roles>>.
+If the {es} {security-features} are enabled, you must have the following 
+privileges:
+
+* `manage_transform` 
```

Comment:
This was `monitor_transform` and `transform_user`. Did you mean to change it to `manage_transform` and `transform_admin`?

Reply:
No, it wasn't intentional. Thanks for spotting it!